flutter/android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.2'
+        classpath 'com.android.tools.build:gradle:4.1.0'
```

Comment:
could be 4.2 but `flutter create android` still uses 4.1 by default so keeping it for compatibility